devicon.json

```
@@ -8199,6 +8199,30 @@
         "color": "#d51c2f",
         "aliases": []
     },
+    {
+        "name": "pug",
+        "altnames": [
+            "pugjs"
+        ],
+        "tags": [
+            "framework",
+            "javascript",
+            "nodejs"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain",
+                "line"
+
+            ],
+            "font": [
+                "plain"
+            ]
+        },
+        "color": "#000000",
```

This should be changed to one of the main colors of the logo.
The main colors of the logo are `#efcca3`, `#56332b`, `#442823` and `#ccac8d`

imo, the best options here are `#efcca3` and `#56332b`
```suggestion
        "color": "#efcca3",
```
```suggestion
        "color": "#56332b",
```
Would like to know your opinion on this @canaleal :)